lib/todo/items_formatter.ex

```diff
@@ -0,0 +1,22 @@
+defmodule ToDo.ItemsFormatter do
+  def items_for_display(items) do
+    add_indexes_to_items_list(items)
+    |> convert_items_list_for_display
+    |> append_to_do_heading
+  end
+
+  def add_indexes_to_items_list(items) do
```

Comment:
The name here describes the implementation. Would a more a generic name be better? (i.e `numbered_items` or `ordered_items`?)